.config/test-config/test-backend-alpha/Dockerfile

```diff
@@ -5,8 +5,8 @@ FROM gravwell/gravwell:latest AS gravwell-stable
 FROM base AS gravwell
 
 RUN apt-get update && apt-get --yes install curl gnupg2 wget
-RUN wget -O /usr/share/keyrings/gravwell.asc https://gin.gravwell.io/debianalphaXYZZY/gin.gravwell.io.gpg.key
-RUN echo 'deb [ arch=amd64 signed-by=/usr/share/keyrings/gravwell.asc ] https://gin.gravwell.io/debianalphaXYZZY community main' > /etc/apt/sources.list.d/gravwell.list
+RUN wget -O /usr/share/keyrings/gravwell.asc https://update.gravwell.io/debian/update.gravwell.io.gpg.key
```

Comment:
Did you mean to change `debianalphaXYZZY` here?

Reply:
The change is right. The key's at a slightly different path on this host, so I had to update it a little differently than the alpha path.